aws/build.gradle

```diff
@@ -162,3 +164,33 @@ def getCurrentGitBranch() {
     }
     return gitBranch
 }
+
+task installSpreadPythonDependency(type: Exec) {
+    workingDir 'src/main/python/spreadsheet_lambda/package'
+    executable 'python3'
+    args '-m', 'pip', 'install', '-r', 'requirements.txt', '-t', '.'
+}
+
+task packageSpreadSheetFunction(dependsOn: [installSpreadPythonDependency], type: Zip) {
+    from fileTree('src/main/python/spreadsheet_lambda/package')
+    from ('src/main/python/spreadsheet_lambda/lambda_function.py')
+    from ('src/main/python/spreadsheet_lambda/credentials.json')
+    from ('src/main/python/spreadsheet_lambda/token.pickle')
+    from ('src/main/python/spreadsheet_lambda/beam-simulation.json')
+    include '**/'
+    archiveName = 'function.zip'
+}
+
+task spreadsheetCreationFunction(dependsOn: packageSpreadSheetFunction, type: AWSLambdaMigrateFunctionTask) {
```

Comment:
Should this have some sort of confirmation since it is something that can affect production lambdas? https://stackoverflow.com/questions/8297008/get-user-choice-before-execution-of-gradle-task

Reply:
I will add it